api/email.py

```diff
@@ -15,8 +15,7 @@
 
 def write_results(results):
     with NamedTemporaryFile(delete=False, mode='w') as tmpfile:
-        writer = DictWriter(tmpfile, results[0].keys())
-        writer.writeheader()
+        writer = csv.writer(tmpfile)
```

Comment:
It will be possible to send a list od dictionaris instead off a list of tuples?
In this way we can get the header from the results

Reply:
I don't think I understand. A _list_ of dictionaries? What do you expect their keys to be? Network names? Those can be retrieved from the query data if needed, which is available in the mongo object.

What is being sent currently is just one alignment of the form
`[('prot1_net1', 'prot1_net2'), ('prot2_net1', 'prot2_net2'), ...]`
that represents protJ_net1 being mapped to protJ_net2. This can be trivially converted to a dictionary by simply calling `dict` on it, but I preferred the list of tuples. The reason for that choice is that if I recall correctly some aligners may misbehave and give partial alignments (like `[..., ('', 'prot3_net2'), ...]`) and that could lead to duplicated keys.

Reply:
I was thinking in something like
```
[{net1: prot1_net1, net2: prot1_net2}, {net1: prot2_net1, net2: prot2_net2}, ...]
```
In this way the api is transparent to the results, so a change in the format results don't need a change in the API

Reply:
I have a few more questions about this format.
1. What if net1 = net2? For instance, the user may want to see if an aligner reports the identity whenever fed with the same network twice.
2. At some point networks may stop being represented purely by a name (think a taxonomy id together with a combined_score threshold for StringDB networks). Do we encode that into a string anyway?
3. I believe that some aligners silently swap their inputs if given in the wrong order (others just fail). What should we do in this case? Stop the user from giving that input? Either way, I should  review which input order does each algorithm assume.
4. Why does this format make the api more transparent to the results than, for example, the list of tuples? In one case it simply assumes that there's a list of tuples and in the other it assumes that there's a list of dictionaries. The header could be included in the list of tuples if needed, or as a separate field in the results object.